lib/core/test/unit/Cardano/Wallet/Primitive/AddressDiscovery/SequentialSpec.hs

```diff
@@ -652,3 +647,7 @@ data Key = forall (k :: Depth -> * -> *).
     , AddressPoolTest k
     ) => Key (Proxy k)
 instance Show Key where show (Key proxy) = show (typeRep proxy)
+
+dummyMnemonic :: SomeMnemonic
+dummyMnemonic = either (error . show) id $
+    SomeMnemonic . entropyToMnemonic @12 <$> mkEntropy (BS.pack $ replicate 16 0)
```

Comment:
Originally seeds of different lengths were used.

Also, matching the 12 with the `16` bytes has to be done manually. We could maybe define a helper:

```Haskell
mkEntropyFromByteStringOfSize @16 $ replicate 16 0
```